parlai/agents/bart/convert_fairseq_to_parlai.py

```diff
@@ -168,6 +168,7 @@ def get_parlai_opt(self) -> Opt:
         # 3. Map other options
         transformer_common_config.update(
             {
+                'datatype': 'valid',  # don't save an optimizer
```

Comment:
This is a little weird as it doesn't seem obvious that saving datatype as valid will always exclude _only_ the optimizer. And it's not obvious that there won't be any other effects.

Is it possible to just explicitly remove the optimizer in `load_fairseq_checkpoint`?

Reply:
I'll do a post vacuum